app/components/equivalence_component/equivalence_component.scss

```diff
@@ -35,3 +35,17 @@
     }
   }
 }
+
+// Used for public display version of equivalence
+// Force the text to match the header height and colours
+// so more readable from a distance
+.equivalence-component.public-displays.horizontal {
+  .trix-content div, div {
+    font-family: $header-font-family;
+    font-weight: $header-font-weight;
+    font-size: $f1;
```

Comment:
Just for future reference, you could have done the following, instead of specifying font-size and line-height:
`@include font(f1);
`